doc/install/installers.rst

```diff
@@ -0,0 +1,213 @@
+.. include:: ../links.inc
+
+MNE-Python installers
+=====================
+
+🛠 About the installers
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Since the release of MNE-Python 1.0, we provide standalone installers for
+MNE-Python. These contain all required dependencies and many additional
+helpful Python packages and tools.
+
+.. hint::
+    **MNE-Python installers are the easiest way to install MNE-Python, and
+    recommended for all users.**
+
+.. note::
+   ⏰ Please understand that due to our limited resources, we currently cannot
+   test on more than the specified platforms. If your favorite platform is
+   missing, you can try to use the installers anyway; however, you'll be on
+   entirely unexplored territory. If you would like us to expand platform
+   support, please get in touch via the forum.
+
+.. warning::
+
+   The installers are very new, and this is the first time we're making them
+   available to a larger group of users. Chances are that you **will**
+   experience some kind of glitches. If you do, please send a report to the
+   forum so we can fix the problem.
+
+🧰 What will be installed
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aside from MNE-Python and all dependencies required for 2D and 3D
+visualization, MVPA ("decoding"), and reading and writing various data
+formats, the following software and Python packages will be installed:
+
+* ``Spyder``, an easy-to-use development environment
+* ``MNE-BIDS`` for BIDS data access
+* ``MNE-Connectivity`` for connectivity analysis
+* ``MNE-FASTER``, a pre-processing pipeline
+* ``MNE-NIRS`` for working with fNIRS data
+* ``MNE-Realtime`` for realtime data analysis
+* ``MNE-Features`` for feature extraction
+* ``MNE-RSA`` for representational similarity analysis
+* ``MNE-Microstates`` for microstates analysis
+* ``MNE-ARI`` for all-resolutions Inference
+* ``autoreject`` for automated artifact rejection
+* ``Jupyter Notebook`` and ``JupyterLab`` for interactive data analysis in the
+  webbrowser
+* ``Pingouin`` for statistics, including repeated-measures ANOVA
+* ``pycircstat`` for circular statistics
+* ``FSLeyes`` for visualization of MRI data
+* ``dcm2niix`` for conversion of DICOM to NIfTI data
+* ``pactools``, ``tensorpac``, ``emd``, ``neurodsp``, ``bycycle``, and
+  ``fooof`` for time-frequency analysis
+* ``NeuroKit2`` for analysis of various biological signals (ECG, EMG, …)
+* ``openneuro-py`` for downloading datasets from OpenNeuro
```

Comment:
These should really all be links accessible in `links.inc`. I can push a change for this and/or any of my other suggestions if you let me know which ones you and @agramfort agree with @hoechenberger 

Also, we need to add `MNE-KIT-GUI` now